src/components/generic/DownloadModal.vue

```diff
@@ -83,7 +83,9 @@
 
     data() {
       return {
-        snippetCopied: false
+        snippetCopied: false,
+        providerString: this.stringify(this.provider),
+        creatorString: this.stringify(this.creator)
```

Comment:
Shouldn't these be computed properties?

Reply:
They do not need to be reactive, so I don't particularly think so.